src/Resources/views/show.xml.twig

```diff
@@ -6,11 +6,11 @@
 <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:image="http://www.google.com/schemas/sitemap-image/1.1" xmlns:xhtml="http://www.w3.org/1999/xhtml">
     {%- for url in url_set -%}
         <url>
-            <loc>{{ url_helper.absolute_or_relative(url.localization, absolute_url) }}</loc>
+            <loc>{{ url_helper.absolute_or_relative(url.location, absolute_url) }}</loc>
             {% if hreflang is not same as(false) and url.alternatives is not empty %}
-                <xhtml:link rel="alternate" hreflang="{{ language_helper.localeToCode(sylius.localeCode) }}" href="{{ url_helper.absolute_or_relative(url.localization, absolute_url) }}"/>
-                {% for locale, location in url.alternatives %}
-                    <xhtml:link rel="alternate" hreflang="{{ language_helper.localeToCode(locale) }}" href="{{ url_helper.absolute_or_relative(location, absolute_url) }}"/>
+                <xhtml:link rel="alternate" hreflang="{{ language_helper.localeToCode(sylius.localeCode) }}" href="{{ url_helper.absolute_or_relative(url.location, absolute_url) }}"/>
+                {% for alternative in url.alternatives %}
```

Comment:
Probably related to an earlier change, but how do we now guard that for each URL there is only 1 alternative per locale? There basically can't be more alternatives for the same locale I think.

Reply:
I'll make a separate issue for this.